crates/pop-contracts/src/templates.rs

```diff
@@ -0,0 +1,142 @@
+// SPDX-License-Identifier: GPL-3.0
+use strum::{
+	EnumMessage as EnumMessageT, EnumProperty as EnumPropertyT, VariantArray as VariantArrayT,
+};
+use strum_macros::{AsRefStr, Display, EnumMessage, EnumProperty, EnumString, VariantArray};
+
+use crate::errors::Error;
+
+#[derive(
+	AsRefStr,
+	Clone,
+	Debug,
+	Default,
+	Display,
+	EnumMessage,
+	EnumProperty,
+	EnumString,
+	Eq,
+	Hash,
+	PartialEq,
+	VariantArray,
+)]
+pub enum Template {
+	/// A minimalist contract template.
+	#[default]
+	#[strum(serialize = "standard", message = "Standard", detailed_message = "Our 'Hello World")]
```

Comment:
The name flipper is well known for those that use ink!. I think it should be mentioned somewhere here.
```suggestion
	#[strum(serialize = "standard", message = "Standard", detailed_message = "ink!'s 'Hello World': Flipper")]
```